lib/db2.js

```diff
@@ -41,7 +42,7 @@ function DB2(settings) {
   this.supportColumnStore = (settings.supportColumnStore || false);
   this.supportDashDB = (settings.supportDashDB || false);
 
-  this.connStr =
+  this.connStrGenerate =
```

Comment:
> is there any particular reason why to expose the generated connection string and also `this.dsn`

thank you @bajtos I agree with you, local variable should be the right solution. will change them.